acceptance/tests/language/sensitive_data_type.rb

```diff
@@ -64,6 +64,13 @@
   sitepp_content = generate_manifest(test_resources)
   assertion_code = generate_assertions(test_resources)
 
+  refutation_resources = test_resources.collect { |assertion_group|
+    refutation_group = assertion_group.clone
+    refutation_group[:assertions] = assertion_group[:assertions].select { |assertion|
+      assertion.has_key?(:refute_match) }
+    refutation_group }
```

Comment:
Ah, good call, thanks.  I just modified the commit in place, now f02cf08, replacing the prior curly braces with do/end blocks.